### PR TITLE
[beta] Move to static.r-l.o stable release

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
r? @pietroalbini or @alexcrichton

### CODE REVIEW

File: src/stage0.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
 
-date: 2019-02-25
+date: 2019-02-27
```
Comment (pietroalbini) [inline src/stage0.txt]:
```suggestion
date: 2019-02-28
```

**Comment by pietroalbini** [comment]
r=me once the release date is fixed (double-checked locally and the one in the suggestion works).

**Comment by pietroalbini** [comment]
It would also be nice to backport the stable release notes (https://github.com/rust-lang/rust/pull/58790) on this PR to avoid wasting more CI resources.

**Comment by Mark-Simulacrum** [comment]
Yep, will fix both of those. I thought I had tested this locally so not sure how I managed to not notice that the date was wrong. Odd.

**Pull request merged**